scripts/compare_bytecode.py

```
@@ -0,0 +1,540 @@
+            if args.seed is not None
+            else random.SystemRandom().randrange(2**32)
+        )
+        rng = random.Random(sample_seed)
+        sample_size = min(args.sample, len(targets))
+        targets = sorted(rng.sample(targets, sample_size), key=lambda item: item[0])
+    if not targets:
+        print("Error: no Python files matched", file=sys.stderr)
+        sys.exit(1)
+
+    report_path = args.output
+    log = lambda *a, **kw: print(*a, file=sys.stderr, **kw)
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)